src/style.scss

```diff
@@ -68,6 +68,9 @@ input {
   position: relative;
   width: 100%;
 
+  min-height: 58px;
+  max-height: 58px;
```

Comment:
What is causing this odd setting versus the 56px / `--auro-size-700` we created to address this issue?